src/Compilers/Test/Utilities/VisualBasic/Extensions.vb

```diff
@@ -344,4 +346,81 @@ Friend Module Extensions
     Friend Function RefKind(this As ParameterSymbol) As RefKind
         Return DirectCast(this, IParameterSymbol).RefKind
     End Function
+
+    <Extension>
+    Friend Function ReduceExtensionMethod(this As MethodSymbol, instanceType As TypeSymbol) As MethodSymbol
+        Return this.ReduceExtensionMethod(instanceType, CompoundUseSiteInfo(Of AssemblySymbol).Discarded)
+    End Function
+
+    <Extension>
+    Friend Function ReduceExtensionMethod(this As MethodSymbol, instanceType As TypeSymbol, proximity As Integer) As MethodSymbol
+        Return this.ReduceExtensionMethod(instanceType, proximity, CompoundUseSiteInfo(Of AssemblySymbol).Discarded)
+    End Function
+
+    <Extension>
+    Friend Function GetUseSiteErrorInfo(this As Symbol) As DiagnosticInfo
+        Return this.GetUseSiteInfo().DiagnosticInfo
+    End Function
+
+    <Extension>
+    Friend Sub Verify(this As ImmutableBindingDiagnostic(Of AssemblySymbol), ParamArray expected As DiagnosticDescription())
+        this.Diagnostics.Verify(expected)
+    End Sub
+
+    <Extension>
+    Friend Sub LookupMember(this As Binder,
+                            lookupResult As LookupResult,
+                            container As NamespaceOrTypeSymbol,
+                            name As String,
+                            arity As Integer,
+                            options As LookupOptions,
+                            <[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo))
+        Dim useSiteInfo As CompoundUseSiteInfo(Of AssemblySymbol) = Nothing
+        this.LookupMember(lookupResult, container, name, arity, options, useSiteInfo)
+        AddDiagnosticInfos(useSiteDiagnostics, useSiteInfo)
+    End Sub
+
+    Private Sub AddDiagnosticInfos(<[In], Out> ByRef useSiteDiagnostics As HashSet(Of DiagnosticInfo), useSiteInfo As CompoundUseSiteInfo(Of AssemblySymbol))
+        If useSiteInfo.Diagnostics IsNot Nothing Then
+            If useSiteDiagnostics Is Nothing Then
+                useSiteDiagnostics = DirectCast(useSiteInfo.Diagnostics, HashSet(Of DiagnosticInfo))
+            Else
+                useSiteDiagnostics.AddAll(useSiteInfo.Diagnostics)
+            End If
+        End If
+    End Sub
+
+    <Extension()>
```

Comment:
>() [](start = 14, length = 2)

Please drop the `()` (or add them everywhere else).

Reply:
> Please drop the () (or add them everywhere else).

This doesn't violate any rules or guidelines.

---
In reply to: [362283395](https://github.com/dotnet/roslyn/pull/40640#discussion_r362283395) [](ancestors = 362283395)

Reply:
Writing code in an internally inconsistent style is a bad programming practice. I am not reviewing only for things we have explicit rules for. I am reviewing for poor programming practices, among other things.

---
In reply to: [362283849](https://github.com/dotnet/roslyn/pull/40640#discussion_r362283849) [](ancestors = 362283849,362283395)